patches/MSBuildLocator/0001-Eliminate-prebuilts.patch

```diff
@@ -20,7 +20,6 @@ index 95ec511..5227a47 100644
    <ItemGroup>
 -    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All"/>
 -    <PackageReference Include="Nerdbank.GitVersioning" Version="3.5.107" PrivateAssets="all" />
-+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="$(MicrosoftSourceLinkGitHubVersion)" PrivateAssets="All"/>
```

Comment:
Why bringing the sourcelink packageref back?

Reply:
Pushed to the wrong branch